tests/src/test/java/alluxio/client/cli/fs/ConfigurationDocGeneratorTest.java

```diff
@@ -141,7 +141,7 @@ public void checkYMLFile() throws Exception {
     String description = pKey.getDescription();
     defaultKeys.add(pKey);
 
-    ConfigurationDocGenerator.writeYMLFile(defaultKeys, mLocation);
+    ConfigurationDocGenerator.handleYMLFile(defaultKeys, mLocation, true);
```

Comment:
add a new parameter for this validate parameter; should be testing both the true and false cases